plaid/institutions_test.go

```diff
@@ -78,8 +82,8 @@ func TestGetInstitutionsByID(t *testing.T) {
 		t.Run(fmt.Sprintf("%#v", options), func(t *testing.T) {
 			// can't use the normal sandbox institution because it only returns the ItemLogins status.
 			institutionID := "ins_12"
-
-			instResp, err := testClient.GetInstitutionByIDWithOptions(institutionID, options)
+			countryCodes := []string{"US"}
+			instResp, err := testClient.GetInstitutionByIDWithOptions(institutionID, countryCodes, options)
 			assert.Nil(t, err)
```

Comment:
perhaps we can add a test to make sure the right error code is returned when we hit the api without an empty country codes array